cowprotocol/eth_flow/.sqlfluff

```diff
@@ -0,0 +1,4 @@
+[sqlfluff:templater:jinja:context]
```

Comment:
all other folders are without _ so I think this one should be ethflow to follow the standard